it/uk/gov/hmrc/agentaccesscontrol/MtdAuthorisationISpec.scala

```diff
@@ -40,9 +43,32 @@ class MtdAuthorisationISpec extends WireMockWithOneServerPerSuiteISpec {
 
         status shouldBe 401
       }
-    }
 
+      "send an AccessControlDecision audit event" in {
```

Comment:
When this test is run the output is a bit odd, it outputs "_when_ send an AccessControlDecision audit event" but we probably want it to output "_should_ send an AccessControlDecision audit event":

```
MtdAuthorisationISpec
[...]
should not grant access
[...]
- when send an AccessControlDecision audit event 
```